### PR TITLE
Frontend routing

### PR DESCRIPTION
**Pull request by hiufungk**
going to pages now redirect back to login page
added logout

### CODE REVIEW

**Review by shchau — APPROVED** [review]
if it works, it works.


for me.

File: frontend/src/components/HigherOrder/withAuth.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import SideBar from "../SideBar";
+import {Redirect} from "react-router-dom";
+import Cookies from 'js-cookie';
+import store from '../../store/index.js';
+
+export default function withAuth(Component, navId) {
+    return class extends Component {
+        render() {
+            let isLoggedIn = store.getState().loginReducers.isLoggedIn || Cookies.get("userPass");
```
Comment (htruong4) [inline frontend/src/components/HigherOrder/withAuth.jsx]: use a const if you are not gonna reassign this let. That way people know that this isn't meant to be reinitialized to another value.

File: frontend/src/pages/Logout.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import {Redirect} from "react-router-dom";
+import Cookies from 'js-cookie';
+import * as LoginActions from "../actions/LoginActions";
+import {connect} from "react-redux";
+
+class Logout extends Component {
+
+    constructor(props) {
+		super(props);
+		Cookies.remove("username");
```
Comment (htruong4) [inline frontend/src/pages/Logout.jsx:11]: Can you reindent lines 10 and 11 to be aligned with the other lines? Just to keep things standardized.

Comment (hiufungk) [reply frontend/src/pages/Logout.jsx:11]: i think this is just git being crazy

File: frontend/src/components/HigherOrder/withAuth.jsx (hunk 1 above)
Comment (hiufungk) [reply frontend/src/components/HigherOrder/withAuth.jsx]: done

**Review by htruong4 — APPROVED** [review]

**Pull request merged**